src/pyHepGrid/src/utilities.py

```diff
@@ -292,7 +292,7 @@ def send(self, tarfile, whereTo, shell=False):
             count +=1
         return success
 
-    def bring(self, tarfile, whereFrom, whereTo, shell=False, timeout = None, suppress_errors=False):
+    def bring(self, tarfile, whereFrom, whereTo, shell=False, timeout = None, suppress_errors=False, force=False):
```

Comment:
`force` is never used here, should it be forwarded to `gfal_copy`?

Reply:
Good spot - yes.  The other arguments are also basically deprecated, I might do something about those later

Reply:
Better to merge this and do that separately, though.